.gitmodules

```diff
@@ -1,6 +1,9 @@
 [submodule "protobufs"]
 	path = protobufs
 	url = https://github.com/meshtastic/protobufs.git
+[submodule "lib/device-ui"]
+	path = lib/device-ui
+	url = https://github.com/meshtastic/device-ui.git
```

Comment:
```suggestion
	url = https://github.com/meshtastic/device-ui.git
    branch = master
```

Reply:
@mariotti:
> As device-ui is basically master driven, I was checking if a submodule can alway point to master. [[comment](https://github.com/meshtastic/firmware/pull/3259#issuecomment-2526297841)]

I believe this solves this particular complaint! But as far as I know you cannot specify a commit hash or tag, so this is good for keeping the HEAD of master but not going deeper. 
@mverch67 I will make a PR — you are welcome to use or reject!

Reply:
I don't think this is a good idea in general as it won't be possible to ever re-build old versions of the software. The branch tag is currently not maintened as there are no automitic builds but will be as soon as the last needed feature is running.

Reply:
Nice! But sadly it will work for a short time :(

If tomorrow we will start versioning the device-ui, which will be the idea, then we have an issue.

I think it is best to leave this to the developers until it is merged.